lib/seattleflu/id3c/cli/command/etl/redcap_det_kiosk.py

```diff
@@ -44,11 +45,15 @@
     "kiosk",
     redcap_url = REDCAP_URL,
     project_id = PROJECT_ID,
-    required_instruments = REQUIRED_INSTRUMENTS,
     revision = REVISION,
     help = __doc__)
 
-def redcap_det_kisok(*, db: DatabaseSession, cache: TTLCache, det: dict, redcap_record: dict) -> Optional[dict]:
+@get_first_redcap_record_instance_only
```

Comment:
There's a convention that decorator names are declarative instead of imperative, since they're a bit of declarative, not imperative, syntax. `get_first_redcap_record_instance_only` is a bit of a mouthful too. What about something like `first_record_instance` or `first_instance_of_record`?

Reply:
Ah, good know. Will update, thanks!